packages/block-library/src/image/editor.scss

```diff
@@ -9,10 +9,6 @@ figure.wp-block-image:not(.wp-block) {
 		opacity: 0.3;
 	}
 
-	figcaption img {
```

Comment:
Nice, the more we can move from editor.scss to style.scss, the better shape we are in.